### PR TITLE
fix(ds): add dialog changes

### PR DESCRIPTION
**Pull request by nistadev**
**Screenshots**

image

image

image

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[UAT. General. Modals. Difference from the design.](https://emurgo.atlassian.net/browse/YOEXT-836)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/components/widgets/Dialog.js [hunk 1]
```diff
@@ -126,34 +136,55 @@ DialogFn.defaultProps = {
...
+    onClick={onClose}
+  >
+    {closeButton || <CrossIcon />}
+  </Box>
+);
+
 const ModalContainer = styled(Box)(({ theme }) => ({
   position: 'relative',
-  minWidth: 'var(--yoroi-comp-dialog-min-width-md)',
+  minWidth: theme.name === 'classic' ? 'var(--yoroi-comp-dialog-min-width-md)' : '612px',
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/widgets/Dialog.js]: This will affect the modern dialogs as well

Comment (lonewold999) [reply packages/yoroi-extension/app/components/widgets/Dialog.js:156]: In general the modern dialogs follow the same styles as revam, at least in all this file, but we can try to add another check if that's an issue.

Comment (nistadev) [reply packages/yoroi-extension/app/components/widgets/Dialog.js]: In general the modern theme follows the revamp styles, at least in this file we can see it all around. If that's an issue we can try do do another check.

**Review by zuzunker — APPROVED** [review]

**Pull request merged**